ktor-server/ktor-server-jetty-jakarta/jvm/src/io/ktor/server/jetty/jakarta/JettyApplicationEngineBase.kt

```diff
@@ -48,8 +54,8 @@ public open class JettyApplicationEngineBase(
      * Jetty server instance being configuring and starting
      */
     protected val server: Server = Server().apply {
-        configuration.configureServer(this)
         initializeServer(configuration)
+        configuration.configureServer(this)
```

Comment:
Porting this change here https://github.com/ktorio/ktor/pull/4367 to Jetty Jarkarta (I did not notice this update in the first instance)